cmd/ltctl/comparison.go

```diff
@@ -161,16 +161,28 @@ func RunComparisonCmdF(cmd *cobra.Command, args []string) error {
 		if err := writeReports(output.Results, outputPath); err != nil {
 			return fmt.Errorf("failed to write reports: %w", err)
 		}
-		printResults(output.Results)
-	}
 
-	if archive {
-		if err := createArchive(outputPath, archivePath); err != nil {
-			return fmt.Errorf("failed to create archive: %w", err)
+		//write to file here
+		resultsFile, errResult := os.Create(filepath.Join(outputPath, "results.txt"))
+		if errResult != nil {
+			fmt.Println("Error:", errResult)
+			return fmt.Errorf("failed to create file: %w", err)
+		}
+		defer resultsFile.Close()
+
+		multiWriter := io.MultiWriter(resultsFile, os.Stdout)
+
+		writeResults(output.Results, multiWriter)
```

Comment:
You have to handle the error here Dylan. Otherwise, there's not much benefit of returning the error :)